### PR TITLE
db/docs-review-improvements

### PR DESCRIPTION
**Pull request by sebastianbochan**

### CODE REVIEW

File: ts/Data/README.md [hunk 1]
```diff
@@ -162,8 +162,8 @@ const chart = new Highcharts.chart('container', {
 
 DataGrid shows and optionally modifies cell content in a table. DataGrid can
 also change the order of cells, but DataTable provides only limited information
-about the original order of a source. Therefor a DataConnector might be needed
-in addition to retrieve the original order.
+about the original order of a source. Therefore, a DataConnector might be needed
```
Comment (bre1470) [inline ts/Data/README.md:165]: There should be no comma.

File: ts/Data/README.md [hunk 2]
```diff
@@ -211,8 +211,8 @@ either an URL or a local source.
 
 ### DataConnector Registry
 
-DataConnector types can be directly loaded via import. In case of bundles
-connectors can also accessed via registry, as the registry gets updated with
+DataConnector types can be directly loaded via import. In case of bundles,
```
Comment (bre1470) [inline ts/Data/README.md:214]: There should be no comma. Instead continue with ...the connectors...

File: docs/dashboards/your-first-dashboard.md [hunk 3]
```diff
@@ -1,24 +1,24 @@
...
 1. Add a div to your webpage with a specific id.
 
     ```html
         <div id="container">
     ```
 
-2. In this basic example we are going to create a dashboard with two cells. One will contain a static HTML and the other will contain a chart created with the Highcharts.
-Thus we have to import the main dashboard package, the Highcharts library and a plugin that allows implementing the Highcharts chart inside the dashboard.
+2. In this basic example we are going to create a dashboard with two cells. One will contain a static HTML and the other will contain a chart created with Highcharts.
+Thus, we have to import the main dashboard package, the Highcharts library and a plugin that allows implementing the Highcharts chart inside the dashboard.
```
Comment (bre1470) [inline docs/dashboards/your-first-dashboard.md:13]: There should be no comma.

File: docs/dashboards/your-first-dashboard.md [hunk 4]
```diff
@@ -85,4 +85,4 @@ Thus we have to import the main dashboard package, the Highcharts library and a
         }]
     ```
 
-6. With that config, you dashboard should look like the one in the [demo](https://jsfiddle.net/gh/get/library/pure/highcharts/highcharts/samples/dashboard/demos/your-first-dashboard).
\ No newline at end of file
+6. With that config, your dashboard should look like the one in the [demo](https://jsfiddle.net/gh/get/library/pure/highcharts/highcharts/samples/dashboard/demos/your-first-dashboard).
```
Comment (bre1470) [inline docs/dashboards/your-first-dashboard.md:88]: There should be no comma.

**Review by bre1470 — CHANGES_REQUESTED** [review]
It would be good to avoid editing data layer in dashboards, as it should go over the master. Some commas er not necessary.

File: docs/dashboards/your-first-dashboard.md (hunk 3 above)
Comment (sebastianbochan) [reply docs/dashboards/your-first-dashboard.md:13]: It's the input from Lars Helge.

File: ts/Data/README.md (hunk 1 above)
Comment (sebastianbochan) [reply ts/Data/README.md:165]: It's the input from Lars Helge.

**Review by pawelfus — APPROVED** [review]
Looks good to me 👍 
Regarding comma-discussions, summoned Lars Helge to join the discussion

**Review by karolkolodziej — APPROVED** [review]

File: ts/Data/README.md (hunk 2 above)
Comment (LB0ge) [reply ts/Data/README.md:214]: I think there should be a comma here, it's more clear. Also, perhaps add a _the_ to the introductory phrase ("In the case of bundles").

File: docs/dashboards/your-first-dashboard.md (hunk 3 above)
Comment (LB0ge) [reply docs/dashboards/your-first-dashboard.md:13]: Same as for _therefore_ at the beginning of a sentence.

File: docs/dashboards/your-first-dashboard.md (hunk 4 above)
Comment (LB0ge) [reply docs/dashboards/your-first-dashboard.md:88]: Comma after the introductory phrase is better in my opinion.

**Pull request merged**